recipes/libxslt/libxslt.inc

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "The XSLT C library for GNOME"
+
+RECIPE_TYPES = "native"
+
+inherit autotools pkgconfig auto-package-utils
```

Comment:
oh right, indeed. I was focused on just getting the tool needed to build the xorg stuff :)